### ISSUE TITLE
How to change path based on user

### ISSUE THREAD

**Issue by swartjie** [alexusmai/laravel-file-manager#252]
Hey Guys,

We're trying to get an ftp file manager which can be used by multiple users at the same time...

So my question is. Can this file manager be used in Laravel but separated out per user?
Meaning if user 1 logs in, it uses user 1's credentials, and if user 2 logs in user 2's?

The main thing being they need to be able to work at the same time too (obviously on different pc's/browsers)?

Kind Regards

**Comment by kieutrongthien** [comment]
I need this feature too, when users logged in, we can set default path for them. 

Example: User A (FolderA), UserB (FolderB). When User A logged in and access file manager, default path is FolderA and they can't back to Root Folder.

**Comment by swartjie** [comment]
I don't think this software is being maintained or supported. 6+ month's and no reply... Sad...

**Comment by iwasherefirst2** [comment]
It's acutally possible. Just create your own `DefaultConfigRepository`, clone from use Alexusmai\LaravelFileManager\Services\ConfigService\DefaultConfigRepository;

and overwrite. 

final public function getLeftPath(): ?string
    {
        return config('file-manager.leftPath');
    }

    /**
     * Default path for right manager
     *
     * null - root directory
     *
     * @return string|null
     */
    final public function getRightPath(): ?string
    {
        return config('file-manager.rightPath');
    }
    
    into the custom path, based on \Auth::user()
